### PR TITLE
Hide keyboard input when clicking on ios current datetime entry

### PR DESCRIPTION
**Pull request by sybers**
When using the wheel mode (iOS 14+), if the user clicks on the currently selected datetime it opens a keyboard over the datetime input.

This behaviour looks weird as we don't know what this number input is for as it overlays with the plugin UI.

This PR disables this behaviour by adding a `.editingDidBegin` event handler which calls `resignFirstResponder` on the datetime ui control.

See below video for demo of the initial issue.

https://user-images.githubusercontent.com/15731884/174600055-4e1f6559-2258-444e-a585-5e94560c1f5a.mov

### CODE REVIEW

File: src/ios/DateTimePicker.m [hunk 1]
```diff
@@ -96,11 +96,12 @@ - (void)initPickerView:(UIView*)theWebView {
...
     
-    // Prefer wheels style on iOS 14.
-    if (@available(iOS 13.4, *)) {
-        datePicker.preferredDatePickerStyle = UIDatePickerStyleWheels;
-    }
-    
+    // Prefer wheels style on iOS 14.
+    if (@available(iOS 13.4, *)) {
+        datePicker.preferredDatePickerStyle = UIDatePickerStyleWheels;
+        [datePicker addTarget:self action:@selector(handleDatePickerTap:) forControlEvents:UIControlEventEditingDidBegin];
```
Comment (skwasjer) [inline src/ios/DateTimePicker.m]: Should this rather be moved to `initPickerView`? This method is called on each 'show', and thus registers the delegate each time, which might be leaky, can you confirm?

**Review by skwasjer — CHANGES_REQUESTED** [review]
Thanks for PR, can you have a look at the requested change?

**Review by skwasjer — APPROVED** [review]

**Comment by skwasjer** [comment]
I had to patch my CI script (see https://github.com/skwasjer/skwas-cordova-plugin-datetimepicker/pull/61), can you rebase/update PR one time to rerun the pipeline? Then I will accept the PR and release it. Thanks!

**Comment by sybers** [comment]
Awesome @skwasjer, thanks for your quick response !

**Pull request merged**